Modules/Control/Field_To_Current.py

```diff
@@ -0,0 +1,9 @@
+import math
+AMOUNT_OF_WRAPS = 100
+RADIUS = 0.09
+
+def Field_To_Current(field):
+    return BaseException/(AMOUNT_OF_WRAPS*math.pi*RADIUS**2)
```

Comment:
What's that supposed to mean? Did you deliberately try to raise a `TypeError: unsupported operand type(s) for /: 'type' and 'float'`?

If so, this exception is just not descriptive of your problem. You should raise instead a `NotImplementedError`, like that:

```
raise NotImplementedError("The function field_to_current in Field_To_Current.py was not yet implemented")
```

Or something like that.

Reply:
Can you explain what it is, please? @shaiTheKimhi